### PR TITLE
Enable MemberImportVisibility check - attempt 2

### PR DESCRIPTION
**Pull request by rnro**
Enable MemberImportVisibility check on all targets. Use a standard string header and footer to bracket the new block for ease of updating in the future with scripts.

See https://github.com/apple/swift-nio/pull/3021

### CODE REVIEW

File: Tests/NIOFileSystemTests/FileInfoTests.swift [hunk 1]
```diff
@@ -23,6 +24,12 @@ import Glibc
 import Android
 #endif
 
+#if canImport(Darwin)
+private let S_IFREG = Darwin.S_IFREG
+#elseif canImport(Glibc)
+private let S_IFREG = Glibc.S_IFREG
+#endif
+
```
[the remark this answers is not in the document]
Comment (finagolfin) [reply Tests/NIOFileSystemTests/FileInfoTests.swift:32]: Thanks, I built this patch natively on Android without a problem, so no changes needed there.

[the remark this answers is not in the document]
Comment (rnro) [reply Tests/NIOFileSystemTests/FileInfoTests.swift:32]: Awesome, thanks

File: Sources/NIOMulticastChat/main.swift [hunk 2]
```diff
@@ -11,6 +11,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
+import CNIOLinux
```
Comment (glbrntt) [inline Sources/NIOMulticastChat/main.swift:14]: This seems like an odd one to need in an example. What symbol is being complained about?

File: Tests/NIOFileSystemTests/FileInfoTests.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (glbrntt) [reply Tests/NIOFileSystemTests/FileInfoTests.swift:32]: I think we'll need the same for Musl as we check in a bunch of other places too.

File: Sources/NIOMulticastChat/main.swift (hunk 2 above)
Comment (rnro) [reply Sources/NIOMulticastChat/main.swift:14]: `addr.address.sin_addr`
```
 94 |             return provider.setIPMulticastIF(addr.address.sin_addr).map { channel }
    |                                                           `- error: property 'sin_addr' is not available due to missing import of defining module 'CNIOLinux'
 ```

File: Tests/NIOFileSystemTests/FileInfoTests.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (finagolfin) [reply Tests/NIOFileSystemTests/FileInfoTests.swift]: > I think we'll need the same for Musl

Tough to say, as the static Musl SDK doesn't let you build the tests, swiftlang/swift-docker#396.

**Review by glbrntt — APPROVED** [review]

**Pull request merged**